roboteam_ai/test/InterfaceTests/MainWindowTest.cpp

```diff
@@ -46,7 +48,7 @@ TEST(MainWindowTest, it_shows_the_visualizer_properly) {
     vis->setShowTactics(true);
     vis->setShowVelocities(true);
 
-    ASSERT_TRUE(vis->showAngles
+    EXPECT_TRUE(vis->showAngles
```

Comment:
instead of making this an && statement I would say run a separate EXPECT_TRUE for all of them, so that you can see which button specifically breaks every time instead of knowing that one of them breaks.